core/src/env/srml/srml_only/impls.rs

```diff
@@ -95,9 +95,6 @@ macro_rules! impl_getters_for_srml_env {
             fn $name() -> $ret_type {
                 unsafe { sys::$ext_name() };
                 Decode::decode(&mut &read_scratch_buffer()[..])
-                    .ok_or(concat!(
-                        stringify!($name), " received an incorrectly sized buffer from SRML"
-                    ))
                     .expect(concat!(
                         stringify!($name), " expects to receive a correctly sized buffer"
                     ))
```

Comment:
Hmm strange what was the semantics before your change? :confused: Good you fixed that!

Reply:
Yeah it just would have meant the first error message was erased. Looks like something that could have happened in a merge conflict resolution.